### PR TITLE
Refactoring for changed file names

### PR DESCRIPTION
**Pull request by gaya3-zipstack**
## What

- Refactoring for changed file names
- Added docstrings
- Addressed pending comments from previous PR

## Why

- To keep the SDK stable for integration

## How

...

## Relevant Docs

-

## Related Issues or PRs

-

## Dependencies Versions / Env Variables

-

## Notes on Testing

Ran unit test cases

## Screenshots

...

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

File: src/unstract/sdk/utils/tool_utils.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 from unstract.sdk.exceptions import FileStorageError
 from unstract.sdk.file_storage import FileStorage, FileStorageProvider
```
Comment (chandrasekharan-zipstack) [inline src/unstract/sdk/utils/tool_utils.py:12]:
```suggestion
from unstract.sdk.file_storage import FileStorage, FileStorageProvider, SharedTemporaryFileStorage
```

File: src/unstract/sdk/utils/tool_utils.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
...
-from unstract.sdk.file_storage.fs_shared_temporary import SharedTemporaryFileStorage
+from unstract.sdk.file_storage.shared_temporary import SharedTemporaryFileStorage
```
Comment (chandrasekharan-zipstack) [inline src/unstract/sdk/utils/tool_utils.py:13]:
```suggestion
```

File: tests/test_fs_permanent.py [hunk 3]
```diff
@@ -6,7 +6,7 @@
 from dotenv import load_dotenv
 
 from unstract.sdk.file_storage import FileStorageProvider
```
Comment (chandrasekharan-zipstack) [inline tests/test_fs_permanent.py:8]:
```suggestion
from unstract.sdk.file_storage import FileStorageProvider, PermanentFileStorage
```

File: tests/test_fs_permanent.py [hunk 4]
```diff
@@ -6,7 +6,7 @@
...
-from unstract.sdk.file_storage.fs_permanent import PermanentFileStorage
+from unstract.sdk.file_storage.permanent import PermanentFileStorage
```
Comment (chandrasekharan-zipstack) [inline tests/test_fs_permanent.py:9]:
```suggestion
```

**Review by chandrasekharan-zipstack — APPROVED** [review]
LGTM, left some minor suggestions

**Review by harini-venkataraman — APPROVED** [review]
LGTM

**Pull request merged**